lib/usage/from.js

```diff
@@ -96,24 +99,75 @@ function options(command, plain) {
     }, {
       text: config.description || ''
     , align: 'left'
-    , width: Math.max(80, width((config.description || '').trim()))
+    , width: 55 // Lines can have line breaks, and wrap automatically.  This can be a low number.
+    })
+
+    if (config.required) {
```

Comment:
This one would probably be better in the main table

```txl
--my_required <string> (required) This flag is required
```

or

```txl
--my_required <string, required> This flag is required
```

Reply:
Yeah, now that I know the columns wrap, this would be better.  You mean just for `required` right?  Having the mutex details as a footer I think is the only way to really describe those things.

Reply:
I think the columns can be changed to:
```
-n --name <type> (required). |. default value | description
```